app/src/main/res/values/strings.xml

```diff
@@ -43,24 +43,24 @@
     <string name="controls_add_to_playlist_title">Add To</string>
     <string name="download_path_title">Video download folder</string>
     <string name="download_path_summary">Downloaded video files are stored here</string>
-    <string name="download_path_dialog_title">Choose the download folder for video files</string>
+    <string name="download_path_dialog_title">Choose download folder for video files</string>
     <string name="download_path_audio_title">Audio download folder</string>
     <string name="download_path_audio_summary">Downloaded audio files are stored here</string>
-    <string name="download_path_audio_dialog_title">Choose the download folder for audio files</string>
+    <string name="download_path_audio_dialog_title">Choose download folder for audio files</string>
     <string name="download_choose_new_path">Change the download folders to take effect</string>
     <string name="autoplay_by_calling_app_title">Autoplay</string>
     <string name="autoplay_by_calling_app_summary">Plays a video when NewPipe is called from another app</string>
     <string name="default_resolution_title">Default resolution</string>
     <string name="default_popup_resolution_title">Default popup resolution</string>
     <string name="show_higher_resolutions_title">Show higher resolutions</string>
-    <string name="show_higher_resolutions_summary">Only some devices support playing 2K/4K videos</string>
+    <string name="show_higher_resolutions_summary">Only some devices can play 2K/4K videos</string>
     <string name="play_with_kodi_title">Play with Kodi</string>
-    <string name="kore_not_found">Kore app not found. Install it?</string>
+    <string name="kore_not_found">Install missing Kore app?</string>
     <string name="kore_package" translatable="false">org.xbmc.kore</string>
     <string name="show_play_with_kodi_title">Show \"Play with Kodi\" option</string>
-    <string name="enable_lock_screen_video_thumbnail_title">Enable lock screen video thumbnail</string>
-    <string name="show_play_with_kodi_summary">Display an option to play a video via Kodi media center</string>
-    <string name="enable_lock_screen_video_thumbnail_summary">When using the background player a video thumbnail will be displayed on the lock screen</string>
+    <string name="enable_lock_screen_video_thumbnail_title">Lock screen video thumbnail</string>
+    <string name="show_play_with_kodi_summary">Displayed option to play a video via Kodi media center</string>
+    <string name="enable_lock_screen_video_thumbnail_summary">Using the background player displays a video thumbnail on the lock screen</string>
```

Comment:
```suggestion
    <string name="enable_lock_screen_video_thumbnail_summary">A video thumbnail will be shown on the lock screen when using the background player</string>
```

I switched the sentence around so that there is more emphasis on the video thumbnail and less emphasis on the background player.

Reply:
But then it becomes a garden path sentence. You have to read to the end to get an idea of what is going on.

Reply:
I think that's the case either way since this sentence has 2 parts that the user needs to understand, so if the user reads only the first half they will be confused but that's the case in your version as well, that is if I stop reading after "Using background player" I as a User will be confused but then why wouldn't I read the whole sentence ? So I restructured the sentence from if(condition) this happens to this happens if (condition) imo perfectly understandable if you don't stop reading midway.

In any case I think that the original line was good too your new version imo makes the sentence slightly awkward unless the purpose was specifically to make it more concise due to size constraints on certain devices with small screens or something like that.

Reply:
I like @cool-student's suggestion. Thanks